PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/FollowCamera.rbxmx

```diff
@@ -148,9 +149,18 @@ local function CreateFollowCamera()
 						end
 					elseif not (isInFirstPerson or userRecentlyPannedCamera) and not VRService.VREnabled then
 						local lastVec = -(self.LastCameraTransform.p - subjectPosition)
+						
 						local y = findAngleBetweenXZVectors(lastVec, self:GetCameraLook())
+						
+						-- This cutoff is to decide if humanoids angle of movement,
+						-- relative to the camera's look vector, is enough that
+						-- we wan the camera to be following them. The point is to provide 
+						-- a sizable deadzone to allow more precision forward movements.
+						local thetaCutoff = 0.4
+						local theta = y/timeDelta
```

Comment:
I am not sure if we check for the 0 timeDelta elsewhere. Sometimes we boot up the script and then run it without a step in between.